stubs/app/Orchid/Screens/User/UserEditScreen.php

```diff
@@ -150,14 +162,23 @@ public function save(User $user, Request $request)
             ->collapse()
             ->toArray();
 
+        $userData = $request->get('user');
```

Comment:
It looks like the $ userData variable is not being used. I think you meant it?

```php
$password = $request->input('user.password');

if (!$user->exists && !empty($password)) {
    $user->password = Hash::make($password);
}
```

Reply:
My fail with last minute changes.
Fixed